src/include/rbd/librbd.hpp

```diff
@@ -75,15 +121,22 @@ namespace librbd {
 
   typedef struct {
     std::string name;
-    int64_t pool;
+    std::string pool;
     group_image_state_t state;
   } group_image_status_t;
 
   typedef struct {
     std::string name;
-    int64_t pool;
+    std::string pool;
```

Comment:
I presume it's also applicable to group_snap_namespace_t in this file, isn't it?